api/lib/ZwaveClient.ts

```diff
@@ -3892,13 +3894,28 @@ class ZwaveClient extends TypedEventEmitter<ZwaveClientEventCallbacks> {
 			const firmwares: Firmware[] = []
 
 			for (const f of files) {
-				const { data, name, target } = f
+				let { data, name } = f
 				if (data instanceof Buffer) {
```

Comment:
@AlCalzone What's the correct way to check buffer here? Should I check if it's an Uint8Array

Reply:
To test whether something is a `Uint8Array`, use the `isUint8Array` function exported from `node:util/types` (not portable) or `@zwave-js/shared` (portable).